ui/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -239,54 +278,56 @@ module.exports = Marionette.LayoutView.extend({
       })
     )
   },
+  getFilterValuesForAttribute(attribute) {
+    return this.filter[attribute]
+      ? this.filter[attribute].map(subfilter => subfilter.value)
+      : []
+  },
+  getCurrentSpecificTypesValue() {
+    const metadataContentTypeValuesInFilter = this.getFilterValuesForAttribute(
+      METADATA_CONTENT_TYPE
+    )
+    const matchTypeAttributeValuesInFilter = this.getFilterValuesForAttribute(
+      getMatchTypeAttribute()
+    )
+    const currentValue = metadataContentTypeValuesInFilter.concat(
```

Comment:
✏️ `const currentValue = [...metadataContentTypeValuesInFilter, matchTypeAttributeValuesInFilter]

Reply:
Ended up replacing the concat + uniq with `union`.